ideas/314-tribute-to-talk.md

```diff
@@ -0,0 +1,69 @@
+TtT Swarm
+=
+
+## Summary and Goals
+
+The Tribute to Talk swarm will deliver SNT based anti-spam mechanisms in accordance with the white paper.
+
+It allows individuals to monetize their time and attention by setting minimum required deposits to message them.
+
+## Contributors
+
+- Eric 
+- Vitaliy 
+- Maciej
+- Tetiana 
+
```

Comment:
Smart contract dev? @richard-ramos ?

Reply:
Most likely, yes. Shouldn't require too much of his time as the contract is already written but still eligible for changes. Working out his time with Iuri.